boards/sparkfun-samd21-mini/Makefile

```diff
@@ -0,0 +1,3 @@
+MODULE = board
```

Comment:
Is this a new board? In that case it should go into seperate PR?

Reply:
I've crossed my branches unfortunately.  Was not supposed to be here apologies.